components/ir/src/commonMain/kotlin/io/github/composegears/valkyrie/ir/util/PathNode.kt

```diff
@@ -166,6 +166,96 @@ private inline fun pathNodesFromArgs(
     }
 }
 
+fun IrPathNode.toPathCommand(): Char = when (this) {
```

Comment:
I moved all this into `sdk:generator:xml` as private functions

https://github.com/ComposeGears/Valkyrie/pull/605

Reply:
sorry missed that one, I've deleted now